packages/api-gateway/src/drivers/http/adapters/places/handlers.js

```diff
@@ -1,15 +1,19 @@
-async function places(req, reply) {
-  const { input } = req.body;
+async function postPlace(req, reply) {
+  const {
+    place_name, price_per_night_usd, host_id, type,
+  } = req.body;
 
-  req.log.info('[http-server]: proving places: ', { input });
+  req.log.info('[http-server]: posting a place');
 
-  await this.places(input);
+  await this.placesService.postPlace({
+    place_name, price_per_night_usd, host_id, type,
+  });
 
   return reply.code(200)
```

Comment:
@Alejandrocuartas as we are creating a new resource we should return the `201` code